From b2fc2a368562d55fadad94d995247bb8cd7e68a3 Mon Sep 17 00:00:00 2001
From: Florian Westphal <fw@strlen.de>
Date: Wed, 24 Oct 2018 12:00:11 +0200
Subject: extensions: limit: unbreak build without libnftnl

Lars Wendler reported 1.8.1 build failure when trying to build without nft backend:

  In file included from ../iptables/nft.h:5, from libxt_limit.c:18: libnftnl/rule.h: No such file or directory

Reported-by: Lars Wendler <polynomial-c@gentoo.org>
Fixes: 02b80972c43 ("ebtables: Merge libebt_limit.c into libxt_limit.c")
Signed-off-by: Florian Westphal <fw@strlen.de>
---
 extensions/libxt_limit.c | 1 -
 iptables/nft-bridge.h    | 1 +
 2 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/extensions/libxt_limit.c b/extensions/libxt_limit.c
index c7b66295..1b324657 100644
--- a/extensions/libxt_limit.c
+++ b/extensions/libxt_limit.c
@@ -15,7 +15,6 @@
 #include <xtables.h>
 #include <linux/netfilter/x_tables.h>
 #include <linux/netfilter/xt_limit.h>
-#include "iptables/nft.h"
 #include "iptables/nft-bridge.h"
 
 #define XT_LIMIT_AVG	"3/hour"
diff --git a/iptables/nft-bridge.h b/iptables/nft-bridge.h
index 9d49ccbe..de52cd71 100644
--- a/iptables/nft-bridge.h
+++ b/iptables/nft-bridge.h
@@ -68,6 +68,7 @@ int ebt_get_mac_and_mask(const char *from, unsigned char *to, unsigned char *mas
 #define EBT_VERDICT_BITS 0x0000000F
 
 struct nftnl_rule;
+struct iptables_command_state;
 
 static const char *ebt_standard_targets[NUM_STANDARD_TARGETS] = {
 	"ACCEPT",
-- 
cgit v1.2.1

